mhkit/wave/graphics.py

```diff
@@ -123,3 +123,143 @@ def plot_matrix(M, xlabel='Te', ylabel='Hm0', zlabel=None, show_values=True,
 
     return ax
 
+def plot_chakrabarti(*args, ax=None):
+    """
+    Plots, in the style of Chakrabart (2005), relative importance of viscous, 
+    inertia, and diffraction phemonena
+
+    Chakrabarti, Subrata. Handbook of Offshore Engineering (2-volume set). 
+    Elsevier, 2005.
+
+    Parameters
+    ------------
+    *args : floats or np.ndarrays or pd.DataFrame
+        If floats: H, L, D
+            where H is wave weight, L is wave length, D is characteristic length
```

Comment:
**wave height**: twice the wave amplitude (what's your question?)

**wave length:** I would've liked to use `lambda`, but this is reserved in python. I could use `Lambda`, but that might be tempting fate.

**units:** everything is dimensionless, so they can be any unit of length as long as they're the same

Reply:
it says H is wave weight :). 

Thanks for the other clarifications. I can make this change, rename L, and check the new commits Monday.

Reply:
@ryancoe we defined [default units](https://mhkit-software.github.io/MHKiT/terminology.html#units) for MHKiT, do the function default should assume MKS units. However, you could add a note to the function that since the KC number is dimensionless one could define H, L, D in a different convention.